alpaca_trade_api/rest.py

```diff
@@ -618,8 +630,10 @@ def _data_get(self,
             else:
                 path = f'/{endpoint_base}/{endpoint}'
                 data['symbols'] = ','.join(symbol_or_symbols)
+            if not endpoint_base or not symbol_or_symbols or not endpoint:
+                path = re.sub(r'(\/)\1+', r'\1', path)  # Remove extra slashes
             resp = self.data_get(path, data=data, api_version=api_version)
-            if isinstance(symbol_or_symbols, str):
+            if isinstance(symbol_or_symbols, str) or endpoint == "news":
```

Comment:
I don't think `endpoint == "news"` is the right abstraction here. I suggest the following approach instead:
- add something like an `Optional[bool]` `resp_grouped_by_symbol` argument to `_data_get` which if None can use the old behaviour (group if a list of symbols were provided, don't group if a single one)
- use that variable here
- for the news endpoint, set it to `False`